lib/libipsec/ipsec_dump_policy.c

```diff
@@ -67,9 +67,9 @@ static char *set_address(char *, size_t, struct sockaddr *);
  * When delimiter == NULL, alternatively ' '(space) is applied.
  */
 char *
-ipsec_dump_policy(caddr_t policy, char *delimiter)
+ipsec_dump_policy(c_caddr_t policy, char const *delimiter)
 {
-	struct sadb_x_policy *xpl = (struct sadb_x_policy *)policy;
+	struct sadb_x_policy const *xpl = (struct sadb_x_policy const *)policy;
```

Comment:
why isn't this `const struct sadb_x_policy *xpl` here and elsewhere.
The project prefers `const type *foo;` rather than `type const *foo;`